lib/tasks/temporary/in_business_data_migration.rake

```diff
@@ -0,0 +1,15 @@
+desc 'Covert in_business to a string'
```

Comment:
This task is in the temporary directory. Are you planning on deleting it? Or is `temporary` a signal that these tasks were run in the past?